### DISCUSSION TITLE
Welcome to dao Discussions!

### DISCUSSION THREAD

**Discussion by andrewsource147** [TeamRaccoons/WAGMI#27] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ConcentricDAO** [comment]
Hi dao discussions community members! My name is Lance Welch with Concentricdao. We are looking for people to join our DAO team forging a new path for Decentralized Autonomous Organizations, bringing Real World, Sustainable Patented Green Energy Technologies that turn wastes and garbage into Highly Valuable Electricity and Abundant Food and Water Commodities, without producing any carbon emissions or pollutions of any kind. In Fact, the Air & Water in and around the Power plants is cleaner coming out, than it was going in!

We are beginners in the blockchain and dao technologies and can use all the help we can get! We have formed our DAO, created our crypto token, and are in need of help through the next steps. We will distribute tokens to anyone who will collaborate and participate with us! 

Is there anyone willing to help us finish making this project into a reality?

Join us in this Noble and Epic Adventure to make Affordable Clean Energy Available to Everyone, Everywhere in the world, as well as the opportunity to own and participate in the Governance, production and profits of this Up and Coming Business!
 
Concentricindustries.net
